README.md

```diff
@@ -225,20 +225,40 @@ Visit https://github.com/MikkelSchubert/adapterremoval/ to get the latest
 version number, and then:
 
 ```
-wget -O adapterremoval-2.3.1.tar.gz \
-     https://github.com/MikkelSchubert/adapterremoval/archive/v2.3.1.tar.gz
-tar xvzf adapterremoval-2.3.1.tar.gz
-cd adapterremoval-2.3.1
+wget -O adapterremoval-2.3.3.tar.gz \
+     https://github.com/MikkelSchubert/adapterremoval/archive/v2.3.3.tar.gz
+tar xvzf adapterremoval-2.3.3.tar.gz
+cd adapterremoval-2.3.3
```

Comment:
Doesn't look like there's anything major here, as you'd expect for a point release: https://github.com/MikkelSchubert/adapterremoval/blob/master/CHANGES.md

I don't think we need to re-run all existing data.

Reply:
Agreed. FWIW, the only reason I upgraded was for the improved error messages.